openpower/custom/package/wqy-zenhei/Config.in

```diff
@@ -0,0 +1,4 @@
+config BR2_PACKAGE_WQY_ZENHEI
```

Comment:
is this being sent to upstream buildroot?

Reply:
yes, just need to get to it